contributing/development/core_and_modules/custom_audiostreams.rst

```diff
@@ -48,69 +48,74 @@ Therefore, playback state must be self-contained in AudioStreamPlayback.
 .. code-block:: cpp
 
     /* audiostream_mytone.h */
-
-    #include "core/reference.h"
-    #include "core/resource.h"
+    
+    #ifndef AUDIOSTREAM_MYTONE_H
+    #define AUDIOSTREAM_MYTONE_H
+    
     #include "servers/audio/audio_stream.h"
-
+    
     class AudioStreamMyTone : public AudioStream {
-        GDCLASS(AudioStreamMyTone, AudioStream)
-
+    	GDCLASS(AudioStreamMyTone, AudioStream);
+    
     private:
-        friend class AudioStreamPlaybackMyTone;
-        uint64_t pos;
-        int mix_rate;
-        bool stereo;
-        int hz;
-
+    	friend class AudioStreamPlaybackMyTone;
+    	uint64_t pos;
+    	int mix_rate;
+    	bool stereo;
+    	int hz;
+    
     public:
-        void reset();
-        void set_position(uint64_t pos);
-        virtual Ref<AudioStreamPlayback> instance_playback();
-        virtual String get_stream_name() const;
-        void gen_tone(int16_t *pcm_buf, int size);
-        virtual float get_length() const { return 0; } // if supported, otherwise return 0
-        AudioStreamMyTone();
-
+    	void reset();
+    	void set_position(uint64_t pos);
+    	virtual Ref<AudioStreamPlayback> instantiate_playback();
+    	virtual String get_stream_name() const;
+    	void gen_tone(int16_t *pcm_buf, int size);
+    	
+    	virtual double get_length() const { return 0; } // if supported, otherwise return 0
```

Comment:
```suggestion
    	virtual double get_length() const { return 0; } // If supported, otherwise return 0.
```
Per the [comment style](https://docs.godotengine.org/en/latest/contributing/development/code_style_guidelines.html#comment-style-guide).